sympy/functions/special/polynomials.py

```diff
@@ -1047,6 +1045,12 @@ def eval(cls, n, x):
             else:
                 return cls._eval_at_order(n, x)
 
+    @classmethod
+    def _eval_at_order(cls, n, x):
+        if n.is_integer and n >= 0:
+            from sympy.polys.orthopolys import hermite_poly
+            return hermite_poly(int(n), _x).subs(_x, x)
```

Comment:
I also don't understand where this has come from

Reply:
Since _ortho_poly cannot be used for import reasons, I had to copy the method.

Maybe it can be done in a better way though?